README.md

```diff
@@ -209,29 +212,29 @@ df.write.options(parameters).avro("/tmp/output")
 
 ```java
 import org.apache.spark.sql.*;
+import org.apache.spark.sql.functions;
 
-SQLContext sqlContext = new SQLContext(sc);
+SparkSession spark = SparkSession.builder().master("local").getOrCreate();
 
 // Creates a DataFrame from a specified file
-DataFrame df = sqlContext.read().format("com.databricks.spark.avro")
-    .load("src/test/resources/episodes.avro");
+Dataset<Row> df = spark.read().format("com.databricks.spark.avro")
+  .load("src/test/resources/episodes.avro");
 
 // Saves the subset of the Avro records read in
-df.filter($"age > 5").write()
-    .format("com.databricks.spark.avro")
-    .save("/tmp/output");
+df.filter(functions.expr("doctor > 5")).write()
```

Comment:
@cloud-fan 

This is a java code example. functions need to be imported explicitly.